packages/types/src/index.ts

```diff
@@ -193,9 +193,9 @@ export type EncodedPolyline = {
 };
 
 export type ElevationData = {
```

Comment:
Keep the old `ElevationData` type, and create a new `ElevationProfile` type with the distance and elevation fields.

Reply:
@binh-dam-ibigroup Hmm... I liked this but we already have an `ElevationProfile` type. It could be `StepElevationProfile` or `ElevationProfileComponent` Since the elevation profile itself is an array of these, the second one might make sense? Although it could be confused with the existing ElevationProfile type which is something else. The first option is okay except that in the GraphQL schema there's no rigid connection between the `Step` type and the `ElevationProfile` type. Let me know what you think...

Reply:
> I liked this but we already have an ElevationProfile type. It could be StepElevationProfile or ElevationProfileComponent [...]

Sure, pick a name that fits.